### ISSUE TITLE
Separate time/date formatting option from currency formatting

### ISSUE THREAD

**Issue by fulldecent** [kimai/kimai#4957]
### Describe the problem

Currently, each user can set a preference for how dates are displayed.

However this setting is intermingled with the other setting for choosing how currency is displayed.

This means it is not possible to choose the standardized ISO 8601 format.

### Describe the solution you'd like

Remove any choice of date formatting and use the international standard ISO 8601 everywhere.

Why? https://xkcd.com/1179/

<img width="392" alt="iso_8601_2x" src="https://github.com/kimai/kimai/assets/382183/f1fb21fc-e034-4628-9fe3-9d4f7e9d976e">

---

Or if there is some very good reason to make this configurable, then at a minimum, each person should be able to specifically choose ISO 8601 date formatting. And this should be the default.

### Screenshots

![a](https://github.com/kimai/kimai/assets/382183/85b9acc6-2ffe-44b7-8a1e-17763251212d)

![Screenshot 2024-07-09 at 11 47 38](https://github.com/kimai/kimai/assets/382183/4dac4a7c-1494-45ef-bbe2-392f4c8adc2b)

**Comment by filcuk** [comment]
I don't agree with the proposal, but I would also like date/time detached from local/currency format.  
Considering how configurable other aspects are, offering choices like 24hr/ampm & common or custom date formats would be ideal.
